docs/configuring-ntfy.md

```diff
@@ -200,6 +200,25 @@ You can define them by adding these variables to your `vars.yml` file (adapt to
 ntfy_visitor_request_limit_exempt_hosts_hostnames_custom: []
 ```
 
+### Expose an endpoint for Prometheus (optional)
+
+You can configure the ntfy server so that it can expose a `/metrics` endpoint for Prometheus.
+
+To expose the endpoint with the default port, add the following configuration to your `vars.yml` file:
+
+```yaml
+ntfy_metrics_default_enabled: true
+```
+
+Alternatively, you can expose the endpoint via a specified IP address and/or port by adding the following configuration to your `vars.yml` file (adapt to your needs). IP address can be omitted.
+
+```yaml
+ntfy_metrics_listen_http_host: 10.0.1.1
```

Comment:
When running in a container, and within a container network, it's probably sensible to default this host to `0.0.0.0`. Looks like it can even be left empty and ntfy will bind on all interfaces (likely `0.0.0.0` and `::` for IPv6), which is even better.

You may be able to use a container-local IP address, but you need to know what container IP address Docker will assign to your container, which is not practical (it's usually a random IP address from its pool).

Unless the port is actually exposed via Docker port publishing (`docker run -p 9090:9090` or `docker run -p SOME_IP:9090:9090`), then you're fine with binding to `0.0.0.0` inside the container and the port won't be accessible from outside.